lib/bundle/locker.rb

```diff
@@ -128,6 +130,15 @@ def mas_list
       end
     end
 
+    def whalebrew_list
+      @whalebrew_list ||= begin
+        Bundle::WhalebrewDumper.images.each_with_object({}) do |image, name_versions|
+          version = `docker image inspect #{image} --format '{{ index .RepoDigests 0 }}' | awk -F':' '{print $2}'`.chomp
```

Comment:
```suggestion
          version = `docker image inspect #{image} --format '{{ index .RepoDigests 0 }}'`.split(':')[1].chomp
```

Reply:
`.second`  or `_, version, =` is nicer than `[1]` imo

Reply:
fair call; i've gone with multiple assignments instead of using an index as I find it easier to read. fixed in 253fb06